src/main/java/io/quarkus/bot/RemoveLabelFromClosedIssue.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.bot;
+
+import io.quarkiverse.githubapp.ConfigFile;
+import io.quarkiverse.githubapp.event.Issue;
+import io.quarkus.bot.config.QuarkusBotConfig;
+import io.quarkus.bot.config.QuarkusBotConfigFile;
+import io.quarkus.bot.util.Labels;
+import org.jboss.logging.Logger;
+import org.kohsuke.github.GHEventPayload;
+import org.kohsuke.github.GHIssue;
+import org.kohsuke.github.GHLabel;
+
+import javax.inject.Inject;
+import java.io.IOException;
+
+public class RemoveLabelFromClosedIssue {
```

Comment:
@gsmet 
Let me know if you want this method to be inside the `TriageIssue` class :)

Reply:
No, it's perfect, as explained elsewhere, the idea is to have a class per rule.